driver-mct/main/seq_diag_mct.F90

```diff
@@ -1391,19 +1400,32 @@ subroutine seq_diag_glc_mct( glc, frac_g, infodata, do_x2g, do_g2x )
 
     if( present(do_x2g))then  ! do fields from coupler to glc (x2g_)
 
-       l2gacc_lx_cnt_avg = prep_glc_get_l2gacc_lx_cnt_avg() ! counter for how many times SMB flux accumulation has occured 
+       l2gacc_lx_cnt_avg = prep_glc_get_l2gacc_lx_cnt_avg() ! counter for how many times SMB flux accumulation has occured
        ic = c_glc_gs
        kArea = mct_aVect_indexRA(dom_g%data,afldname)
        lSize = mct_avect_lSize(x2g_g)
 
        do n=1,lSize
           ca_g =  dom_g%data%rAttr(kArea,n)*g2x_g%rAttr(index_g2x_Sg_icemask,n)
           nf = f_wgsmb; budg_dataL(nf,ic,ip) = budg_dataL(nf,ic,ip) + ca_g*x2g_g%rAttr(index_x2g_Flgl_qice,n)
+          if (index_x2g_Fogx_qiceli /= 0) then
+             ! qiceli and qicehi are positive into glc
+             nf = f_wpolar
+             budg_dataL(nf,ic,ip) = budg_dataL(nf,ic,ip) + ca_g*x2g_g%rAttr(index_x2g_Fogx_qiceli,n)
```

Comment:
```suggestion
             budg_dataL(nf,ic,ip) = budg_dataL(nf,ic,ip) - ca_g*x2g_g%rAttr(index_x2g_Fogx_qiceli,n)
```
The sum of the area-weighted `qiceli` is negative in the coupler history files. Since I assume the area-weighted melt rate consistent with melting, we need a negative sign here so that latent heat enters the glc component.